os/net/routing/routing.h

```diff
@@ -90,7 +90,7 @@ struct routing_driver {
   /**
    * Returns the global IPv6 address of a source routing node
    *
-   * \param ipaddr A pointer where to copy the IP address of the node
+   * \param addr A pointer where to copy the IP address of the node
```

Comment:
for consistency, would be better to change the parameter name, not the docs

Reply:
🤦‍♂️ Fixed!